src/actions/slack/test_utils.ts

```diff
@@ -241,6 +249,40 @@ describe(`slack/utils unit tests`, () => {
                 },
             ]).and.notify(done)
         })
+
+        it("returns correct users", (done) => {
+            const slackClient = new WebClient("token")
+            const result = getDisplayedFormFields(slackClient, "manual")
```

Comment:
Lots of stubbing in the previous test, but none here.  I have no clue what's going on or what this is trying to test, since I don't see any users in the expected result for a test titled "returns correct users" 😅

Reply:
Also, I'm super confused as to why the results on line 135 (`const result = getDisplayedFormFields(slackClient, "channels")`) wouldn't be expected to contain items labeled "Comment" or "Filename", since from the code, those seem to be always pushed onto the response. 

Maybe I'm not understanding what exactly `chai.expect(result).to.eventually.deep.equal()` does

Reply:
fixed the test name